### PR TITLE
fail safe for missing file

### PR DESCRIPTION
**Pull request by dforstercon**
Duplicated failsafe check from not alias version to this function.
May also needed in other branches.

### CODE REVIEW

**Comment by GawainLynch** [comment]
> May also needed in other branches.

We cascade our branch merges, so no stress there.

File: src/Twig/Handler/ImageHandler.php [hunk 1]
```diff
@@ -271,6 +271,10 @@ private function getThumbnailUri(Thumbnail $thumb)
      */
     private function getAliasedUri($filename, $alias)
     {
+        if ($filename == null){
```
Comment (GawainLynch) [inline src/Twig/Handler/ImageHandler.php]: Can you make the comparison strict please

**Comment by dforstercon** [comment]
I changed the comparison to strict. 
But as I said, I just copied the failsafe from the getThumbnailUri function, which is non-strict either. :)

**Comment by GawainLynch** [comment]
Yeah, we have a few of those … I'll look deeper in the morning, but the defaults on the two feeder functions are nulls, and we're trying to rid the code base of loose junk.

Thanks very much for the effort :+1:

**Comment by GawainLynch** [comment]
I had a look back through the git history, and the `getThumbnailUri()` comparison was deliberately loosened for BC.

This should be GTG however.

**Pull request merged**